runtime/src/bank.rs

```diff
@@ -4938,6 +4954,23 @@ impl Bank {
             .round() as u64
     }
 
+    // Calculate cost of loaded accounts size in the same way heap cost is charged at
+    // rate of 8cu per 32K. Citing `program_runtime\src\compute_budget.rs`: "(cost of
+    // heap is about) 0.5us per 32k at 15 units/us rounded up"
+    //
+    // Before feature `support_set_loaded_accounts_data_size_limit_ix` is enabled, or
+    // if user doesn't use compute budget ix `set_loaded_accounts_data_size_limit_ix`
+    // to set limit, `compute_budget.loaded_accounts_data_size_limit` is set to default
+    // limit of 64MB; which will convert to (64M/32K)*8CU = 16_000 CUs
+    //
+    fn calculate_loaded_accounts_data_size_cost(compute_budget: &ComputeBudget) -> u64 {
+        const KILOBYTE: u64 = 1024;
+        (compute_budget.loaded_accounts_data_size_limit as u64)
+            .saturating_add((32_u64 - 1).saturating_mul(KILOBYTE))
+            .saturating_div(32_u64.saturating_mul(KILOBYTE))
+            .saturating_mul(compute_budget.heap_cost)
+    }
```

Comment:
```suggestion
        const ACCOUNT_DATA_COST_PAGE_SIZE: u64 = 32.saturating_mul(1024);
        (compute_budget.loaded_accounts_data_size_limit as u64)
            .saturating_add(ACCOUNT_DATA_COST_PAGE_SIZE.saturating_sub(1))
            .saturating_div(ACCOUNT_DATA_COST_PAGE_SIZE)
            .saturating_mul(compute_budget.heap_cost)
    }
```

we need to do the mul to 32k _before_ the sub of one so that when the caller requests one byte, we're pushed up to the first page. as is there won't be any cost until the at least one _kilobyte_ is requested

Reply:
Yes yes! thanks for spotting it